fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -3628,9 +3841,15 @@ struct ShardedRocksDBKeyValueStore : IKeyValueStore {
 
 	CoalescedKeyRangeMap<std::string> getExistingRanges() override { return shardManager.getExistingRanges(); }
 
+	void logRecentRocksDBFlushStats() override { return flushListener->logRecentRocksDBFlushStats(); }
+	void logRecentRocksDBCompactionStats() override { return compactionListener->logRecentRocksDBCompactionStats(); }
+
 	std::shared_ptr<ShardedRocksDBState> rState;
 	rocksdb::Options dbOptions;
 	std::shared_ptr<RocksDBErrorListener> errorListener;
+	std::shared_ptr<RocksDBWriteStallListener> writeStallListener;
+	std::shared_ptr<RocksDBFlushListener> flushListener;
+	std::shared_ptr<RocksDBCompactionListener> compactionListener;
```

Comment:
They could be merged into the same listener class.

Reply:
Yes. I can merge those. Do we want to merge writeStallListener and flushListener and compactionListener into the RocksDBErrorListener? (change RocksDBErrorListener to RocksDBEventListener)